### PR TITLE
Signup NUX: new prologue

### PR DESCRIPTION
**Pull request by nheagy**
Refs #8109 

This adds the new sign up buttons to the login prologue.

prologue methods

Notes:
- Signup with Google will be a future PR
- Uses the same `NUXButtonsView` as other parts of the new NUX
  - changed to allow top and bottom button to controller as white/blue 
  - changed buttons to have minimum 50px height. This is what the prologue buttons were before, but we may want @folletto to ok making them all match

To test:
- Try to the new prologue "Sign up" UI
- Make sure the old flow works with the feature flag off

### CODE REVIEW

**Comment by folletto** [comment]
> changed buttons to have minimum 50px height. This is what the prologue buttons were before, but we may want @folletto to ok making them all match

The most important thing there is for them to match, so ok for me. :)

**Comment by folletto** [comment]
> changed buttons to have minimum 50px height

I did an internal check and on iOS might be better to align to iOS standard of 44px if it's simple enough to be done in this PR :)

**Comment by nheagy** [comment]
@folletto 44px is not the standard, it's just the minimum. For example, on an action sheet the buttons are about 57px tall. I think 50px is a good size, and 44px looked too narrow.

**Comment by nheagy** [comment]
@ScoutHarris ok, conflicts with develop have been merged. Should be ready for ya!

File: WordPress/Classes/ViewRelated/NUX/LoginPrologueSignupMethodViewController.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        configureButtonVC()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
+    private func configureButtonVC() {
+        guard let buttonViewController = buttonViewController else {
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/NUX/LoginPrologueSignupMethodViewController.swift]: Looking at the design, there are a couple of differences:
1. The designed buttons have a blue text color.
2. The designed buttons have email and google icons.

screen shot 2018-02-15 at 11 43 11 am

Comment (folletto) [reply WordPress/Classes/ViewRelated/NUX/LoginPrologueSignupMethodViewController.swift]: Color: I thought I did it matching the other, but anyway consistency > blueprint. So let's go for consistency, we can review later all of them.

Icon: I think it's mandatory for Google Login ([ref](https://developers.google.com/identity/branding-guidelines)).

My take: if this is ready, ship it and open a separate one to do these changes.

**Comment by ScoutHarris** [comment]
Changed my review to Approved per @folletto 's comment:

> if this is ready, ship it and open a separate one to do these changes.

@nheagy - :shipit: !

**Pull request merged**